src/components/testimonial.js

```diff
@@ -0,0 +1,32 @@
+import React from "react";
+import Carousel from 'react-bootstrap/Carousel'
+
+const Testimonial = () =>(
+    <div className="container-fluid mt-5 mb-5" id="testimonial">
```

Comment:
@faithghlee : Just to make it clean change this to `<Container fluid className="mt-5 mb-5">` so that we only use react-bootstrap components. 😄